lane_detection_node: 
  ros__parameters: 
    Hue_low : 37 
    Hue_high : 77 
    Saturation_low : 75 
    Saturation_high : 255 
    Value_low : 49 
    Value_high : 255 
    number_of_lines : 1 
    error_threshold : 0.05 
    Width_min : 0 
    Width_max : 500 
    gray_lower : 61 
    inverted_filter : 0 
    kernal_size : 5 
    erosion_itterations : 2 
    dilation_itterations : 2 
    crop_width_decimal : 1.0 
    rows_to_watch_decimal : 1.0 
    rows_offset_decimal : 0.01 
    camera_centerline : 0.5 
mraysu_node: 
  ros__parameters: 
    Hue_low : 37 
    Hue_high : 77 
    Saturation_low : 75 
    Saturation_high : 255 
    Value_low : 49 
    Value_high : 255 
    number_of_lines : 1 
    error_threshold : 0.15 
    Width_min : 0 
    Width_max : 500 
    gray_lower : 61 
    inverted_filter : 0 
    kernal_size : 5 
    erosion_itterations : 2 
    dilation_itterations : 2 
    crop_width_decimal : 1.0 
    rows_to_watch_decimal : 1.0 
    rows_offset_decimal : 0.01 
    camera_centerline : 0.5 
lane_guidance_node: 
  ros__parameters: 
    Kp_steering : 0.9 
    Ki_steering : 0.0 
    Kd_steering : 0.4 
    zero_throttle  : 0.0 
    max_throttle : 1
    min_throttle : 0.5
    error_threshold : 0.03
    max_right_steering : 1.0 
    max_left_steering : -1.0 
vesc_twist_node: 
  ros__parameters: 
    max_rpm : 2000 
    steering_polarity : 1 
    throttle_polarity : 1 
    zero_throttle  : 0.0 
    max_throttle : 1
    min_throttle : 0.5
    max_right_steering : 1.0 
    straight_steering : 0.0 
    max_left_steering : -1.0 
adafruit_twist_node: 
  ros__parameters: 
    steering_polarity : 1 
    throttle_polarity : 1 
    zero_throttle  : 0.0 
    max_throttle : 0.3
    min_throttle : 0.05
    max_right_steering : 1.0 
    straight_steering : 0.0 
    max_left_steering : -1.0 
